01.data_download.py

```
@@ -0,0 +1,141 @@
+# import package needed
+# %pip install -r requirements.txt
+
+#%%
+# import package needed
+import os
+import tarfile
+import wget
+import json
+from urllib.request import urlopen
+import pandas as pd
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)